### PR TITLE
Integrated React to the Client part of the project

### LINKED ISSUE #6: add react and webpack
integrate react and webpack by following the guide line in readme file

### PR DESCRIPTION
**Pull request by kendevops**
## Description
I implemented the react app and such that all the Linting and tests are still available.

Fixes #6 

## How Has This Been Tested?
I integrated the react app to the client part without removing the lints and i ran yarn start and the app was working on port 3000

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have added necessary inline code documentation
- [x ] I have added tests that prove my fix is effective and that this feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,13 +14,16 @@
...
   },
   "author": {
     "name": "Build For SDG",
     "email": "buildforsdg@andela.com"
   },
-  "homepage": "https://github.com/BuildForSDG/js-starter#readme",
+  "homepage": "",
   "repository": {
     "type": "git",
     "url": "https://github.com/BuildForSDG/js-starter.git"
```
Comment (tonymj76) [inline package.json:32]: change this to our github repo

File: package.json [hunk 2]
```diff
@@ -17,13 +14,16 @@
     "test:watch": "jest --watch",
     "test:cover": "jest --coverage",
     "lint": "eslint \"src/**/*.js\"",
-    "start": "parcel ./src/index.html"
+    "start": "react-scripts start",
```
Comment (tonymj76) [inline package.json:20]: you are missing a lot of config.. like babel, webpack and hot reload to make this easy

File: package.json [hunk 3]
```diff
@@ -17,13 +14,16 @@
     "test:watch": "jest --watch",
     "test:cover": "jest --coverage",
     "lint": "eslint \"src/**/*.js\"",
```
Comment (tonymj76) [inline package.json:19]: eslint should have it rules files

File: package.json [hunk 4]
```diff
@@ -1,11 +1,8 @@
...
+  "name": "good_health",
+  "version": "0.1.0",
   "description": "A boilerplate for starting js code",
-  "keywords": [
-    "js-starter",
-    "boilerplate"
-  ],
+  "keywords": ["js-starter", "boilerplate"],
   "private": false,
   "main": "dist/index.js",
```
Comment (tonymj76) [inline package.json]: this should be in the webpack config file

**Review by tonymj76 — CHANGES_REQUESTED** [review]
please do the following changes

File: src/app.js [hunk 5]
```diff
@@ -1,25 +1,12 @@
...
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <React.Fragment>
```
Comment (tonymj76) [inline src/app.js:5]: why are you using react fragment is unnecessary, addition of extra component can lead to bulky code base

File: src/index.js [hunk 6]
```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
```
Comment (tonymj76) [inline src/index.js:4]: this is one of the reason i say we should not use create react app cli tool, now that you removed service worker from the app, what happens if one day we all decide to use service worker or server side rendering? then we will have to inject and start a fresh configuration.. which you know can lead to problem

**Review by tonymj76 — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 5 above)
Comment (kendevops) [reply src/app.js:5]: They are a necessity to control the Dom tree of the components

Comment (tonymj76) [reply src/app.js:5]: brother we don't need fragment in this code let all write simply code

**Review by tonymj76 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**